pkgs/data/misc/mailcap/default.nix

```diff
@@ -1,23 +1,23 @@
 { lib, fetchzip }:
 
 let
-  version = "2.1.48";
+  version = "2.1.49";
 
 in fetchzip {
   name = "mailcap-${version}";
 
   url = "https://releases.pagure.org/mailcap/mailcap-${version}.tar.xz";
-  sha256 = "08d0avz8971hkggd60dk9yyd14izz24yag3prpfafbvm670jlmqg";
+  sha256 = "1nl6zv6lwjyg85m7ffxlfy9gmhicg76l7iiy3qnn2vhkmaxipmcy";
 
   postFetch = ''
     tar -xavf $downloadedFile --strip-components=1
     substituteInPlace mailcap --replace "/usr/bin/" ""
-    gzip mailcap.4
+    gzip mailcap.5
```

Comment:
Isn't this done automatically?

Reply:
I will check. I merely adapted the existing recipe.

Reply:
```suggestion
```

Reply:
I spent some time trying to get it fixed but it would require rewriting the entire expression for which I currently have no time. Apparently fetchzip (which is used) doesn't like installManPage and friends.